packages/mdc-tabs/README.md

```diff
@@ -426,3 +539,75 @@ Updates the active tab to be the tab at the given index, emits `MDCTabBar:change
 #### MDCTabBarFoundation.getActiveTabIndex() => number
 
 Returns the index of the currently active tab.
+
+
+## Tab Bar Scroller
+
+### Tab Bar Scroller component API
+
+#### Properties
+
+| Property Name | Type | Description |
+| --- | --- | --- |
+| `tabBar` | `MDCTabBar` | _(read-only)_ The scroller's tab bar. |
+
+#### MDCTabBarScroller.layout() => void
+
+Proxies to the foundation's `layout()` method.
+
+### Using the Foundation Class
+
+MDC Tab Bar Scroller ships with an `MDCTabBarScrollerFoundation` class that external frameworks and libraries can use to integrate the component. As with all foundation classes, an adapter object must be provided.
+
+
+### Adapter API
+
+| Method Signature | Description |
+| --- | --- |
+| `addClass(className: string) => void` | Adds a class to the root element. |
+| `removeClass(className: string) => void` | Removes a class from the root element. |
+| `eventTargetHasClass(target: HTMLElement, className: string) => boolean` | Returns true if target has a given class name |
+| `addClassToForwardIndicator(className: string) => void` | Adds a given class to the forward indicator |
+| `removeClassFromForwardIndicator(className: string) => void` | Removes a given class from the forward indicator |
+| `addClassToBackIndicator(className: string) => void` | Adds a given class to the back indicator |
+| `removeClassFromBackIndicator(className: string) => void` | Removes a given class from the back indicator |
+| `isRTL() => boolean` | Returns true if in RTL context. False otherwise. |
+| `registerBackIndicatorClickHandler(handler: EventListener) => void` | Registers an event handler to be called when a `click` event happens on the back indicator |
+| `deregisterBackIndicatorClickHandler(handler: EventHandler) => void` | Deregisters an event handler from a `click` event happening on the back indicator |
+| `registerForwardIndicatorClickHandler(handler: EventHandler) => void` | Registers an event handler to be called when a `click` event happens on the forward indicator |
+| `deregisterForwardIndicatorClickHandler(handler: EventHandler) => void` | Deregisters an event handler from a `click` event happening on the forward indicator. |
+| `registerTabInteractionHandler(evt: string, handler: EventHandler) => void` | Registers an event handler to be called when a `focus`, `touchstart`, or `mousedown` event happens on the root of the component. These events gets dispatched to the listener during the capture phase. They also govern the scrolling behavior when tabs are tabbed to or actioned on. |
```

Comment:
I think it makes sense to keep some mention of tab in the name. To us, since we anticipate this may have something to do with the act of pressing the tab _key_, there may be some cognitive baggage. For everyone else I think they'll assume the semantics here are describing a tab component, and any interaction associated with it.